pkg/visor/visorconfig/v1.go

```diff
@@ -53,13 +53,13 @@ type Dmsgpty struct {
 
 // Transport defines a transport config.
 type Transport struct {
-	Discovery         string          `json:"discovery"`
-	AddressResolver   string          `json:"address_resolver"`
-	PublicAutoconnect bool            `json:"public_autoconnect"`
-	TransportSetup    []cipher.PubKey `json:"transport_setup_nodes"`
-	LogStore          *LogStore       `json:"log_store"`
-	StcprPort         int             `json:"stcpr_port"`
-	SudphPort         int             `json:"sudph_port"`
+	Discovery           string          `json:"discovery"`
+	AddressResolver     string          `json:"address_resolver"`
+	PublicAutoconnect   bool            `json:"public_autoconnect"`
+	TransportSetupNodes []cipher.PubKey `json:"transport_setup_nodes"`
```

Comment:
I would avoid node here because of two reasons:

- easy to confuse with `setup node`
- we opted to name setup node as setup node because it is running a visor instead of just an HTTP API

Reply:
any naming convention that makes it obvious that it takes a public key is ok with me.
`TransportSetupPKs`?

Reply:
this was resolved in another PR I believe